src/Umbraco.Tests/App.config

```diff
@@ -54,62 +54,46 @@
 
   <runtime>
       <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
-          <dependentAssembly>
```

Comment:
Any reason why we are getting rid of the C# Roslyn Analyzers here or is it just bloat that we don't need?!